### PR TITLE
clean up git init

### PR DESCRIPTION
**Pull request by jquense**

### CODE REVIEW

File: GitUtilities.js [hunk 1]
```diff
@@ -8,7 +8,18 @@ function hasTag(tag) {
     .then(() => true, () => false);
 }
 
-exports.init = dest => execa('git', ['init'], { cwd: dest, stdio: 'inherit' });
+exports.isGitRepo = dest =>
```
Comment (taion) [inline GitUtilities.js]: what is this?

you could also just test for the existence of a `.git`

Comment (jquense) [reply GitUtilities.js]: i could but i figured this was a bit more robust if your in a nested folder

File: GitUtilities.js [hunk 2]
```diff
@@ -8,7 +8,18 @@ function hasTag(tag) {
...
+  execa('git', ['rev-parse'], { cwd: dest, stdio: 'inherit' })
+    .then(() => true)
+    .catch(() => false);
+
+exports.init = dest =>
+  exports
+    .isGitRepo(dest)
+    .then(
+      isGit =>
+        isGit && execa('git', ['init'], { cwd: dest, stdio: 'inherit' }),
```
Comment (taion) [inline GitUtilities.js]: wait actually i'm confused... you want to run `init` if it's already a git repo?

Comment (jquense) [reply GitUtilities.js]: ooops

**Pull request merged**